src/app/static/src/tests/components/accessibility.test.ts

```diff
@@ -60,6 +61,58 @@ describe(`accessibility`, () =>{
 
         const paragraph = rendered.find("#accessibility-content").findAll("p")
         expect(paragraph.at(11).text()).toBe("Cette déclaration a été préparée " +
-            "le 24 février 2021. Elle a été mise à jour pour la dernière fois le 24 février 2021.")
+            "le 24 février 2021. Elle a été mise à jour pour la dernière fois le 26 février 2021.")
+    })
+
+    it(`renders accessibility purpose list tags as expected in French`, () => {
+        const store = createStore();
+        const rendered = shallowMount(Accessibility, {store});
+        rendered.vm.$store.state.language = Language.fr
+
+        const li = rendered.find("#accessibility-content").find("#purpose").findAll("ul li")
+        expect(li.length).toBe(3)
+        expect(li.at(0).text()).toBe("Redimensionner votre fenêtre avec le contenu reformaté de manière appropriée")
+        expect(li.at(1).text()).toBe("Ajustez la taille de votre texte sans que le site ne devienne moins utilisable")
+        expect(li.at(2).text()).toBe("Lisez facilement le texte grâce au contraste suffisant entre les éléments de premier plan et d'arrière-plan")
+    })
+
+    it(`renders accessibility example list tags as expected in French`, () => {
```

Comment:
You can test English and French translations at the same time with `expectTranslated` to keep tests of the same logic together

Reply:
This isn't possible because the component does not have it's own store.